scripts/build-and-test.sh

```diff
@@ -54,8 +60,7 @@ if [[ $VARIANT == "coverage" ]]; then
   jobs
   # Kill it gracefully
   kill -INT %1
-  sleep 1
-  kill -INT %1 || echo "Dead already"
+  wait
```

Comment:
Noting that, in an updated version, I added a sleep of 5 seconds to let the server `wind down` before issuing a kill. Not sure if it's needed or it will help ?